src/pages/XhainWayToSchool/index.tsx

```diff
@@ -1,78 +1,88 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 import styled from 'styled-components';
 import { ArticleHeader, ArticleWrapper, Intro } from '~/components2/Article';
 import LogoFile from './assets/wappen.png';
 import LogoFile2 from './assets/wappen@2x.png';
 import LogoFile3 from './assets/wappen@3x.png';
-import { Section01, Section02, Section03 } from './sections';
+import {
+  Section01Network,
+  Section02Surroundings,
+  Section03Crossing,
+  Section04Next,
+} from './sections';
 
 const Wrapper = styled(ArticleWrapper)`
-  // background: none;
-  // h2 {
-  //   text-transform: none;
-  // }
+  background: none;
+  h2 {
+    text-transform: none;
+  }
 
-  // .contentWrapper {
-  //   box-shadow: none;
-  // }
+  .fmc-article-tocentry {
+    margin-bottom: 0.75em;
+    font-family: 'Roboto Slab';
+    font-weight: 100;
+  }
 
-  // .fmc-article-tocentry {
-  //   margin-bottom: 0.75em;
-  //   font-family: 'Roboto Slab';
-  //   font-weight: 100;
-  // }
-
-  // .fmc-article-tocentry-active {
-  //   font-weight: 700;
-  // }
+  .fmc-article-tocentry-active {
+    font-weight: 700;
+  }
 `;
 
 const Logo = styled.img`
   height: 42px;
 `;
 
 const XhainWayToSchool = () => (
-  <Wrapper
-    hasToc
-    tocTitle="Inhalt"
-    enumerateToc={false}
-    bannerTitle="Schulwegsicherheit in XHain"
-    logo={
-      <Logo
-        src={LogoFile2}
-        srcSet={`${LogoFile} 1x, ${LogoFile2} 2x, ${LogoFile3} 3x`}
-        alt=""
+  <>
+    <Helmet>
+      <title>
+        Schulwegsicherheit in XHain – Mit Unterstützung von FixMyCity
+      </title>
+      <meta
+        name="description"
+        content="Das Bezirksamt Friedrichshain-Kreuzberg möchte sichere Schulwege für
+        alle Schüler*innen schaffen. Als Grundlage für die nötigen Maßnahmen wurde diese Analyse zum Schulwegenetz erstellt."
```

Comment:
What is the general way of writing gender? We have way more words using the colon `:`

Reply:
I assumed that in this case we don't apply our own standard but the one by Xhain which is given via the wording doc. Since the wording doc is signed of, I would not change it…